src/xrGame/Actor.cpp

```diff
@@ -83,7 +83,7 @@ const float respawn_auto = 7.f;
 extern float cammera_into_collision_shift;
 extern int g_first_person_death;
 
-string32 ACTOR_DEFS::g_quick_use_slots[4] = {NULL, NULL, NULL, NULL};
+string32 ACTOR_DEFS::g_quick_use_slots[4] = {{'\0'}, {'\0'}, {'\0'}, {'\0'}};
```

Comment:
If I'm not mistaken, this should be enough:
```suggestion
string32 ACTOR_DEFS::g_quick_use_slots[4] = {};
```